src/pages/ja/guides/cms.md

```diff
@@ -0,0 +1,38 @@
+---
+title: AstroでCMSを利用する
+description: CMSを利用してコンテンツをAstroへ追加する
+layout: ~/layouts/MainLayout.astro
+setup: |
+  import CMSGuidesNav from '~/components/CMSGuidesNav.astro';
+i18nReady: true
+---
+
+**ヘッドレスCMSをAstroプロジェクトにつなぐ準備はしましたか？** ここでは、CMSを導入するための方法を紹介します。
+
+## CMS ガイド
+
+<CMSGuidesNav />
+
+これらのページの多くはスタブページです。あなたの貢献を待っているリソースのコレクションです。
+
+## なぜCMSを使うか
+
+コンテンツ管理システムはAstroプロジェクト外でコンテンツの投稿・アセットの管理をすることが出来ます。
+
+これによって、コンテンツを扱う新しい機能を使えるようになります。ほとんどのCMSはビジュアルコンテンツエディター、標準的なコンテンツの種類を指定する機能、他の人との共同作業などの機能を提供しています。
+
+CMSは特定の構造に従ったコンテンツで利用でき、多くの場合はダッシュボードのような操作性とWYSIWYGエディターツールを提供します。
+CMSはマークダウンファイルの代わりとなるCMSのリッチテキストエディターを使ってブログ記事を書くことが出来ます。もしくは、CMSを利用してEコマースショップの商品リストを管理して、不完全なリストを除外するために特定のフィールドを必須項目とすることが出来ます。
+
+AstroプロジェクトはCMSからコンテンツを取得して、自分のサイト上の好きな場所に好きなように表示することができます。
+
+
+## Astroと相性の良いCMS
+
+Astroはコンテンツの_表現_に気を払うので、上記リストにある_ヘッドレス_CMSを選んだほうが良いです。つまり、CMSはコンテンツ作成を補助しますが、コンテンツを表示するサイトを生成することはありません。代わりにAstroプロジェクトがコンテンツデータをフェッチして利用します。
+
+StoryblokのようないくつかのヘッドレスCMSは、Astroのサイトへ特殊なコンテンツをフェッチできるようにAstroの[integration](/en/guides/integrations-guide/)を提供しています。他のものは自分でインストールしてリモートコンテンツを取得するために使用する JavaScript SDK を提供しています。
```

Comment:
We use the `/ja/` link even for pages that haven’t been translated yet to avoid moving a user out of their own language:

```suggestion
StoryblokのようないくつかのヘッドレスCMSは、Astroのサイトへ特殊なコンテンツをフェッチできるようにAstroの[integration](/ja/guides/integrations-guide/)を提供しています。他のものは自分でインストールしてリモートコンテンツを取得するために使用する JavaScript SDK を提供しています。
```